website/docs/Pipelines/reblock_gvcf_spec.md

```diff
@@ -0,0 +1,68 @@
+* No more MIN_DP -- joint DP varies?
+* InbreedingCoeff values will vary due to some low quality genotypes being rounded down to GQ0
+
+#### Cost/scale improvements
+
+* Storage footprint is reduced compared with HaplotypeCaller output
+* Fewer VariantContexts, i.e. lines, per VCF speeds up GenomicsDB/Hail import
+* Fewer alternate alleles reduces memory requirements for merging
+
+
+### Specific improvements compared with 7-band schema
+* Not dropping GQ0s -- reblocked GVCF should cover all positions (that input GVCF covers)
+* No overlaps -- only overlapping positions should be two variants, i.e. deletions, on separate haplotypes
+(No more no-calls in GVCFs-- all genotypes should be called, positions with no data will be hom ref with GQ0; NOTE that this is not being validated and GenotypeGVCFs outputs will have no-calls to preserve previous behavior)
+* Latest Terra WDL and config at https://app.terra.bio/#workspaces/ccdg-gatk-supp-terra/CCDG_WES_Reblocking_Work_F3/workflows/ccdg-gatk-supp-terra/ReblockGVCF-gatk4_exomes_goodCompression_fixed (NOTE that this has an off-by-one issue for contigs starts of exactly 1, which we don’t see in Broad production)
+
+## CCDG WGS (7 bands)
+* Suitable for de novo calling from exomes
+* Arguments -drop-low-quals -do-qual-approx --floor-blocks -GQB 10 -GQB 20 -GQB 30 -GQB 40 -GQB 50 -GQB 60
+Q* UALapprox output in INFO field, required for GnarlyGenotyper
+* Low quality alt alleles will be dropped (below stand-call-conf = 30)
+* Alleles that aren’t called will be dropped 
+* homRef calls at GQ0 (default) are dropped
+* Non-ref AD zeroed out
+* GTs with non-ref are converted to GQ0 homRef and dropped
+* Blocks are floored -- GQ output for block is the lower bound for that block, regardless of median or min
+* Current WDL at https://portal.firecloud.org/?return=terra#methods/methodsDev/ReblockGVCF-gatk4_exomes_goodCompression/4
+* As used for UKBB
+
+
+### gnomADv3 schema (deprecated -- 2 bands)
+Arguments -drop-low-quals -do-qual-approx -rgq-threshold 10
+
+* Default GQ blocks, < 20 and >= 20
+* homRef calls below GQ10 are dropped
+* → remaining tranche of homRef calls between GQ 10 and 20 gnomAD team had to filter
+* QUALapprox output in INFO field, required for GnarlyGenotyper
+* Low quality alt alleles will be dropped (below stand-call-conf = 30)
+* Extra INFO annotations dropped, only data required for VQSR retained: DP, MQ, ReadPosRankSum, MQRankSum (strand bias data is in FORMAT)
+* Non-ref AD zeroed out
+* WDL at https://portal.firecloud.org/?return=terra#methods/eric-methods/ReblockGVCF-gatk4/1/wdl
+
+## Original HaplotypeCaller schema (current Warp single-sample pipeline)
```

Comment:
no longer 'current' - WARP single sample prior to v3.0.0, right?

Reply:
Also maybe worth noting that this is without reblocking, this is the standard output from haplotype caller?